scripts/build-os-packages/nfpm.yaml

```diff
@@ -0,0 +1,44 @@
+name: ggshield
+section: utils
+vendor: GitGuardian
+maintainer: GitGuardian <dev@gitguardian.com>
+license: MIT
+homepage: https://github.com/GitGuardian/ggshield
+description: |-
+  Detect leaked secrets and other potential security vulnerabilities
```

Comment:
```suggestion
  Detect leaked secrets and other potential security vulnerabilities.
```

Reply:
The lack of dot at the end is intentional, see https://github.com/GitGuardian/ggshield/pull/878#discussion_r1572348024

Reply:
sorry !